### PR TITLE
Add support for version 4.03

### PR DESCRIPTION
**Pull request by jlorentson**

### CODE REVIEW

File: src/test/java/com/imsweb/geocoder/GeocodingInputTest.java [hunk 1]
```diff
@@ -35,7 +36,7 @@ public void testBasicInput() {
         assertEquals("false", queryParams.get("notStore"));
         assertEquals("true", queryParams.get("census"));
         assertEquals("allAvailable", queryParams.get("censusYear"));
-
+        assertEquals("true", queryParams.get("ShouldDoExhaustiveSearch"));
```
Comment (ctmay4) [inline src/test/java/com/imsweb/geocoder/GeocodingInputTest.java]: Shouldn't all the query params use camelCase?  Others do, but there are a couple that do not.  I think we should be consistent.

**Review by ctmay4 — COMMENTED** [review]
Just the one comment

**Comment by ctmay4** [comment]
@criderp you might want to review this as well.

File: src/test/java/com/imsweb/geocoder/GeocodingInputTest.java (hunk 1 above)
Comment (jlorentson) [reply src/test/java/com/imsweb/geocoder/GeocodingInputTest.java]: D'oh. I agree.

**Review by ctmay4 — APPROVED** [review]

**Pull request merged**